### PR TITLE
Updated Readme to use correct references. Added link to virtual package

### PR DESCRIPTION
**Pull request by kstkn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
 HTTP Factories
 ==============
 
-This is the implementation of [PSR-17 (HTTP Message Factories)][psr-17]. Please refer to the
-specification for a description.
+This repository holds all interfaces related to [PSR-17 (HTTP Message Factories)][psr-17]. 
+Please refer to the specification for a description.
+
+You can find implementations of the specification by looking for packages providing the 
+[psr/http-factory-implementation](https://packagist.org/providers/psr/http-factory-implementation) virtual package.
```
Comment (Jean85) [inline README.md:8]: Maybe we can point the link directly to the dependents of that package?

Comment (kstkn) [reply README.md:8]: Why to the dependents? From my perspective, visitor of this repository will mostly be interested in implementations.

Comment (Jean85) [reply README.md:8]: Uh sorry, I misread the link! That is the correct one!

**Review by Jean85 — APPROVED** [review]

**Review by shadowhand — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+ª
```
Comment (shadowhand) [inline README.md]: What is this line for?

Comment (kstkn) [reply README.md]: Oh, I'm sorry, that was an accident

**Comment by shadowhand** [comment]
Thanks!

**Comment by shadowhand** [comment]
Tagged in 1.0.1.

**Pull request merged**